doc/source/tune/_tutorials/tune-distributed.rst

```diff
@@ -148,12 +148,12 @@ If a string is provided, then it must include replacement fields ``{source}`` an
         MyTrainableClass,
         name="experiment_name",
         sync_config=tune.SyncConfig(
-            sync_to_driver=custom_sync_func
+            syncer=custom_sync_func
         )
     )
 
 When syncing results back to the driver, the source would be a path similar to ``ubuntu@192.0.0.1:/home/ubuntu/ray_results/trial1``, and the target would be a local path.
-This custom sync command is used to restart trials under failure. The ``sync_to_driver`` is invoked to push a checkpoint to new node for a paused/pre-empted trial to resume.
+This custom sync command is used to restart trials under failure. The ``syncer`` is invoked to push a checkpoint to new node for a paused/pre-empted trial to resume.
```

Comment:
I am a little confused by this statement. Is this only talking about sync to driver case?
In sync to cloud case, a checkpoint is "pulled" into the trainable, right?
Should we describe in a more generic way?

Reply:
I revised this whole section in the latest commit to make it easier to understand